crates/uv-interpreter/src/interpreter.rs

```diff
@@ -314,6 +315,18 @@ impl Interpreter {
             },
         }
     }
+
+    /// Check if the given Python version is satisfied by this interpreter.
+    ///
+    /// If a patch version is present, we will require an exact match.
+    /// Otherwise, just the major and minor version numbers need to match.
+    pub fn satisfies(&self, version: &PythonVersion) -> bool {
```

Comment:
Inverse of `PythonVersion::is_satisfied_by(... Interpreter)` due to the dependency reversal